build.gradle

```diff
@@ -842,6 +842,62 @@ tasks.create(name: "jarConnect", dependsOn: connectPkgs.collect { it + ":jar" })
 
 tasks.create(name: "testConnect", dependsOn: connectPkgs.collect { it + ":test" }) {}
 
+project(':server') {
+  archivesBaseName = "kafka-server"
+
+  dependencies {
+    implementation project(':clients')
+    implementation project(':server-common')
+
+    implementation libs.slf4jApi
+
+    compileOnly libs.log4j
+
+    testImplementation project(':clients').sourceSets.test.output
+
+    testImplementation libs.mockitoCore
+    testImplementation libs.junitJupiter
+    testImplementation libs.slf4jlog4j
+  }
+
+  task createVersionFile() {
+    def receiptFile = file("$buildDir/kafka/$buildVersionFileName")
```

Comment:
am not a gradle expert but my IDE is telling me that $buildDir is deprecated and replaced by `getLayout().getBuildDirectory()` instead

Reply:
I used the same pattern used by other modules. We can replace all of them via a separate PR.

Reply:
ok

Reply:
Fyi :

"Note that, at this stage, Gradle will not print deprecation warnings if you still use Project.buildDir. We know this is a big change and want to give time for authors of major plugins to move away from its usage first."